pypika/tests/test_functions.py

```diff
@@ -400,6 +400,21 @@ def test__substring(self):
 
         self.assertEqual("SELECT SUBSTRING(\"foo\",2,6) FROM \"abc\"", str(q))
 
+    def test__raw__sql_on_select(self):
+        q = Q.from_(self.t).select(fn.Raw('current_date'))
+
+        self.assertEqual("SELECT current_date FROM \"abc\"", str(q))
```

Comment:
For this syntax, where there are no parentheses, you should use the `PseudoColumn` class.

```
CurrentDate = pypika.PseudoColumn('current_date')
Q.from_(self.t).select(CurrentDate)
# --> SELECT current_date FROM "abc"
```

Reply:
Uhh, nice think, we don't have this in the documentation. May I create a note about this there?

Reply:
yeah there are a few minor features that are missed in the docs. If you would like to add something in there, it would most likely be very helpful for the next person who needs it.